packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -13,33 +13,40 @@ import { DashboardState, SaveableDashboard } from '../../store/state';
 import { PickRequiredOptional, RecursivePartial } from '../../types';
 import { SiteWiseQuery } from '@iot-app-kit/source-iotsitewise';
 import { DashboardMessages, DefaultDashboardMessages } from '../../messages';
+import { ClientContext } from './clientContext';
 
 import '@cloudscape-design/global-styles/index.css';
 import '../../styles/variables.css';
+import { IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
 
 export type IotDashboardProps = {
   messageOverrides?: RecursivePartial<DashboardMessages>;
   query?: SiteWiseQuery;
   onSave?: (dashboard: SaveableDashboard) => void;
+  client?: IoTSiteWiseClient;
```

Comment:
is it possible to derive query from this client? The 2 are related in some way. I know we are going to clean up the multi client situation eventually, just want to note it

Reply:
Yes, I think we need to do that next. Just trying to add the minimum required right now to source the client from being passed in, per Brian's comment.